src/pykiso/test_coordinator/test_case.py

```diff
@@ -183,7 +183,9 @@ def setUpClass(cls) -> None:
         """
         options = get_logging_options()
         if options.report_type == "junit":
-            initialize_logging(None, options.log_level, options.report_type)
+            initialize_logging(
```

Comment:
I'm wondering if it would also be possible to run this in the XmlTestResult to achieve the same without having this hack, to be determined later :)